README.md

```diff
@@ -16,9 +16,20 @@ Run `make` to build `./bin/firehose.so`. Then use with Fluent Bit:
 ./fluent-bit -e ./firehose.so -i cpu \
 -o firehose \
 -p "region=us-west-2" \
--p "delivery-stream=example-stream"
+-p "delivery_stream=example-stream"
```

Comment:
Had to push a new revision to rectify the README. For both Linux and Windows, the parameter is `delivery_stream` while it was mentioned as `delivery-stream` which would have resulted in plugin not being initialised.